### DISCUSSION TITLE
<feat> : Add Today's Popular Articles

### DISCUSSION THREAD

**Discussion by Devanshukoli** [dailydotdev/daily#842] [Feature Requests]
I want to suggest a feature, that lets folks know the **Today's popular blogs/articles**  in the **Discover** section. Because when we open our browser the articles are many.

 And for folks like me that want to know what people are interested in reading articles, which tech type they prefer, and where the frequently search for it.

**Comment by idoshamun** [comment]
Hi @Devanshukoli,

Thanks for the suggestion. Just to be clear, you want a date filter in the popular feed to see only today's posts, right?

**Reply by Devanshukoli** [reply]
Hey, 
@idoshamun Thanks for checking out my questions.

Just to be we are both on the same page, I will elaborate quickly.
- A date filter that let me know, **what is popular, for a particular day**
- And by **Particular day**, I mean whenever users want they can find out on **xx-xx-xxxx** day this feed (listing of that feed) will show.

I feel that's a lot to ask, But @daily.dev really helping me a lot in my learning journey. I read blogs/articles that pick my domain of interest more regularly thanks to it. And it also helped me in building my recent business project.
This one was really helpful `https://blog.devgenius.io/my-client-website-got-hacked-lessons-learned-and-steps-to-strengthen-website-security-a37fda9e1080`

Thanks.
